internal/oci/oci.go

```diff
@@ -436,3 +437,20 @@ func (r *Runtime) RestoreContainer(ctx context.Context, c *Container, sbSpec *rs
 
 	return impl.RestoreContainer(ctx, c, sbSpec, infraPid, cgroupParent)
 }
+
+func (r *Runtime) CreateNamespaces(ctx context.Context, c *Container) (string, error) {
```

Comment:
It's possible we may want to define a sandbox level interface that runtimePod also satisfies, and we can have a separate interface for oci and VM (where pinns is called). that may be cleaner from a code organization perspective